src/main/webapp/app/components/calendarButton/UsageAnalysisCalendarButton.tsx

```diff
@@ -0,0 +1,123 @@
+import React, { useState } from 'react';
+import {
+  InputGroup,
+  Button,
+  Dropdown,
+  DropdownButton,
+  Row,
+  Form,
+  Col,
+  Container,
+} from 'react-bootstrap';
+import classnames from 'classnames';
+import { getMomentInstance } from 'app/shared/utils/Utils';
+import DayPickerInput from 'react-day-picker/DayPickerInput';
+import 'react-day-picker/lib/style.css';
+import moment from 'moment';
+import {
+  APP_LOCAL_DATETIME_FORMAT_Z_FORCE,
+  APP_LOCAL_DATE_FORMAT,
+} from 'app/config/constants';
+import MomentLocaleUtils from 'react-day-picker/moment';
+import { DefaultTooltip } from 'cbioportal-frontend-commons';
+import pluralize from 'pluralize';
+
+type CalendarButtonProps = {
+  currentDate: string | undefined;
+  currentFromDate: string | undefined;
+  currentToDate: string | undefined;
+  fromDate: (newDate: string) => void;
+  toDate: (newDate: string) => void;
+  filterToggled: (filterActive: boolean) => void;
+};
+
+const DATE_PLACEHOLDER = 'Select Date';
+
+export const UsageAnalysisCalendarButton: React.FunctionComponent<CalendarButtonProps> = props => {
```

Comment:
- [ ] Can we do not close the window after selecting the From?
- [ ] We should indicate somehow when the calendar filter is selected. Maybe change the background to dark blue like when you hover over?